runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/jobsubmission/JobServerDriver.java

```diff
@@ -96,8 +96,11 @@ protected InMemoryJobService createJobService() throws IOException {
 
     @Option(
         name = "--sdk-worker-parallelism",
-        usage = "Default parallelism for SDK worker processes (see portable pipeline options)")
-    private Long sdkWorkerParallelism = 1L;
+        usage =
+            "Default parallelism for SDK worker processes (see portable pipeline options). "
```

Comment:
Consider slightly different order: `Parallelism level for SDK worker processes. 
This value is used only when the pipeline option sdkWorkerParallelism is set to 0, otherwise the value provided in pipeline options takes precedence. Default is 0, which means worker parallelism will be dynamically decided by a runner. See also: sdkWorkerParallelism Pipeline Option`. Also, should last sentence be a Javadoc reference?

Reply:
Makes sense. I will update the wording.